lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm

```diff
@@ -133,17 +133,11 @@ sub get_levels {
 
 sub determine_genome_build {
   my $self = shift;
-  my $ref_build_name1 = shift;
-  my $genome_build = "";
-  my $b37_build =
-    Genome::Model::Build::ReferenceSequence->get(name => "GRCh37-lite-build37");
-  my $ref_build1 = Genome::Model::Build::ReferenceSequence->get(name => $ref_build_name1);
-  if($b37_build->is_compatible_with($ref_build1)) {
-    return "b37";
-  } else {
+  my $supplied_build_name = shift;
+  my $igv_name = Genome::Model::Tools::Analysis::ResolveIgvReferenceName::resolve_igv_reference_name($supplied_build_name) ||
```

Comment:
When calling into other modules but not using the `->`, we need to `use` them first to avoid a warning from the autoloader.